### DISCUSSION TITLE
Save credentials without having a persistent file storage

### DISCUSSION THREAD

**Discussion by TempleClause** [bbernhard/signal-cli-rest-api#377] [Q&A]
Hello there,

I was wondering if there is an easy way to get this library to work without a persistent file storage.

Would like to run it on GCP Cloud Run which is stateless. Is there an easy way to make this work without having a persistent file storage? 

Thanks

**Comment by bbernhard** [comment]
This project is just a small REST API wrapper around [signal-cli](https://github.com/AsamK/signal-cli), so all the heavy lifting is done in signal-cli. If signal-cli supports it, it can also be exposed via this project. But I doubt, that this will be (easily) possible.
